### ISSUE TITLE
Master branch tests fails

### ISSUE THREAD

**Issue by stabenfeldt** [backstopmedia/bleeding-edge-sample-app#52]
I'm working from the **master** branch. I guess that's the right place to start?

I discovered that nothing happens when I press Submit on the Survey builder.
So I decided to run the tests:
```javascript
PhantomJS 2.0.0 (Mac OS X): Executed 6 of 77 SUCCESS (0 secs / 0.04 secs)
ERROR [karma]: [TypeError: Line must be greater than or equal to 1, got 0]
TypeError: Line must be greater than or equal to 1, got 0
  at SourceMapConsumer_findMapping [as _findMapping] (/Users/martins/Work/ReactJS/bleeding-edge-sample-app/node_modules/karma/node_modules/source-map/lib/source-map/source-map-consumer.js:286:15)
  at SourceMapConsumer_originalPositionFor [as originalPositionFor] (/Users/martins/Work/ReactJS/bleeding-edge-sample-app/node_modules/karma/node_modules/source-map/lib/source-map/source-map-consumer.js:3
46:24)
  at /Users/martins/Work/ReactJS/bleeding-edge-sample-app/node_modules/karma/lib/reporter.js:49:28
  at String.replace (native)
  at /Users/martins/Work/ReactJS/bleeding-edge-sample-app/node_modules/karma/lib/reporter.js:36:23
  at /Users/martins/Work/ReactJS/bleeding-edge-sample-app/node_modules/karma/lib/reporters/base.js:105:14
  at Array.forEach (native)
```

Am I doing anything wrong or is it a bug in the master branch?

Cheers,
Martin

**Comment by stabenfeldt** [comment]
Hi, 

Can I get feedback on this issue? I want to believe that the book is worth reading.  When the master branch fails and I get no feedback I get the feeling that this is a half ready product. 

Please let me know why the examples fail and how to fix them.

**Comment by eiriklv** [comment]
+ 1

Would be great to have a working demo..

**Comment by karlmikko** [comment]
Closed: see #49
